DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/JobProfileWhatYouWillDo/Index.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@model DFC.Digital.Web.Sitefinity.JobProfileModule.Mvc.Models.JobProfileWhatYouWillDoViewModel
+
+@if (Model.IsWhatYouWillDoCadView)
+{
+    <section class="job-profile-subsection" id="@Html.Raw(Model.SectionId)">
+        <h2 class="heading-large">@Html.Raw(Model.Title)</h2>
+        @if (Model.IsIntroActive && !string.IsNullOrWhiteSpace(Model.Introduction))
+        {
+            <p>@Model.Introduction</p>
+        }
+        <h3>@Model.DailyTasksSectionTitle</h3>
```

Comment:
Are we sure this is correct mark-up? I would have thought this would be a sub section `<section />`?
```
<section id="WhatYouWillDo">
      <h2 class="heading-large">What you will do?</h2>
      <p>intro</p>
      <section class="job-profile-subsection" id="DayToDayTasks">
            <h3>Daily Tasks</h3>
            ....
      </section>
      <section class="job-profile-subsection" id="WorkEnvironment">
            <h3>Work environment</h3>
            ...
      </section>
</section>
```